database.go

```diff
@@ -1176,6 +1194,7 @@ func (db *database) Measurements() Measurements {
 	for _, m := range db.measurements {
 		measurements = append(measurements, m)
 	}
+	sort.Sort(measurements)
```

Comment:
I probably shouldn't have added that.  No need to impose that performance penalty on all callers.